Source/Core/Core/PowerPC/JitArm64/JitArm64_RegCache.h

```diff
@@ -265,9 +262,12 @@ class Arm64FPRCache : public Arm64RegCache
 
 	// Returns a guest register inside of a host register
 	// Will dump an immediate to the host register as well
-	ARM64Reg R(u32 preg);
+	ARM64Reg R(u32 preg, bool only_lower = true);
```

Comment:
"RegType type = REG_LOWER_PAIR" as second argument, so we can extent it.
Do you think it's wise to not use the safest way (=false)? Or we shouldn't provide a default value, so it's done explicit within every instruction